packages/solr.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# The following script is pulling the binary files from the
+# recommended mirror site. Visit http://www.apache.org/dyn/closer.lua/lucene/solr/
+# for a full list of available mirrors.
+#
+# Remember to set the necessary version of Java. See
+# https://codeship.com/documentation/languages/java-and-jvm-based-languages/
+# for more information.
+#
+# To run this exact configuration, add the following to your builds:
+curl -sSL https://raw.githubusercontent.com/codeship/scripts/master/packages/solr.sh | bash -s
+
+# Set Java version
+jdk_switcher home oraclejdk8
+jdk_switcher use oraclejdk8
+
+# Install solr
+SOLR_VERSION="6.0.0"
+SOLR_MIRROR_URL="http://mirror.olnevhost.net/pub/apache/lucene/solr"
+CACHED_DOWNLOAD="${HOME}/cache/solr-${SOLR_VERSION}.zip"
+
+rm -rf ~/.solr
+mkdir ~/.solr
```

Comment:
I'd personally move the installation to `${HOME}/solr`, that's easier to debug on a SSH Debug build, as you don't have to look for a hidden directory.